File: VB/datainspectorExport/App_Start/DashboardConfig.vb
Imports Microsoft.VisualBasic
Imports DevExpress.DashboardWeb
Imports DevExpress.DashboardWeb.Mvc
Imports System.Web.Routing

Namespace datainspectorExport
	Public NotInheritable Class DashboardConfig
		Private Sub New()
		End Sub
		Public Shared Sub RegisterService(ByVal routes As RouteCollection)
			routes.MapDashboardRoute("dashboardControl")

			Dim dashboardFileStorage As New DashboardFileStorage("~/App_Data/Dashboards")
			DashboardConfigurator.Default.SetDashboardStorage(dashboardFileStorage)

			DashboardConfigurator.Default.SetConnectionStringsProvider(New DevExpress.DataAccess.Web.ConfigFileConnectionStringsProvider())

			Dim dataSourceStorage As New DataSourceInMemoryStorage()
			DashboardConfigurator.Default.SetDataSourceStorage(dataSourceStorage)
		End Sub

	End Class
End Namespace

File: CS/datainspectorExport/App_Start/DashboardConfig.cs
using DevExpress.DashboardWeb;
using DevExpress.DashboardWeb.Mvc;
using System.Web.Routing;

namespace datainspectorExport {
    public static class DashboardConfig {
        public static void RegisterService(RouteCollection routes) {
            routes.MapDashboardRoute("dashboardControl");

            DashboardFileStorage dashboardFileStorage = new DashboardFileStorage("~/App_Data/Dashboards");
            DashboardConfigurator.Default.SetDashboardStorage(dashboardFileStorage);

            DashboardConfigurator.Default.SetConnectionStringsProvider(new DevExpress.DataAccess.Web.ConfigFileConnectionStringsProvider());
            
            DataSourceInMemoryStorage dataSourceStorage = new DataSourceInMemoryStorage();            
            DashboardConfigurator.Default.SetDataSourceStorage(dataSourceStorage);
        }
        
    }
}